inception-doc/src/main/resources/META-INF/asciidoc/user-guide/getting-started.adoc

```diff
@@ -1,79 +1,399 @@
+
+- Agreement
+
+- Evaluation
+
+|Project Manager |- annotate +
+- courate +
+- create projects +
+- add new documents +
+- add guidelines +
+- manage users +
+- open documents like an other user sees it (read only) |All pages
+available
+|===
+
+* In the section **Layers, **you may modify existing layers to use for
+your annotation and create own layers. In case you do not want to work
+on default layers only, we highly recommend reading the main
+documentation on layers. LINK EINFÜGEN
+* In the section **Knowledge Base, **you can change the settings for the
+knowledge bases used in your project and also import local and remote
+knowledgebases or create a new one. Herefore, just click the
+„create“-button and follow the wizzard.
+** Note that you can have several knowledge bases in your INCEpTION
+instance but you can choose for every project which one(s) to use. Using
+many knowledge bases in one project will slow down the performance.
+** Via the *Dashboard* (click the Dashboard-button at the top centre),
+you can get to the knowledge base page. This is a page different from
+the one in the project settings where you can modify and work on your
+knowledge bases.
+** For details on knowledge bases, see our main documentation LINK
+EINFÜGEN or our tutorial video „Overview“ which also talks about
+knowledge bases quickly:
+https://www.youtube.com/watch?v=wp4AN3p23mQ&list=PL5Hz5pttaj96SlXHGRZf8KzlYvpVHIoL-&index=3&t=0s..
+If you like to explore a knowledge base learning by doing, you may
+download and import the example project _Interactive Concept Linking_:
+https://inception-project.github.io/example-projects/concept-linking/
+which contains a small knowledge base.
+* In the section **Recommenders, **you can create and modify your
+recommenders. They learn from your behaviour and give automatic
+suggestions for creating annotations. For details on this topic to broad
+for a getting-started-guide, see our main documentation LINK EINFÜGEN or
+our tutorial video „Recommender Basics“:
+https://www.youtube.com/watch?v=Xz3Hs8Lyoeg&list=PL5Hz5pttaj96SlXHGRZf8KzlYvpVHIoL-&index=3
+.
+
+* In the *Guidelines* section, you may import guideline files for your
+team to read and to check quickly while working. On the annotation page
+(→ _dashboard_ → _annotation_ → open any document), everyone can quickly
+look them up by clicking on the guidelines button on the top (it looks
+like a book).
+
+* We have imported a whole project already (the example project) as well
+as single documents (in the **documents **section). In the **Export
+**section, you can export your project partially or wholly. Projects
+which have been exported can be imported again in INCEpTION the way we
+did before: at the start page with the *Import* button. We recommend
+exporting projects on a regular basis in order to have a backup. For the
+different formats, their strengths and weaknesses, check our main
+documentation LINK EINFÜGEN. We recommend using WebAnno TSV x.x (where
```

Comment:
We recommend using AUTO

Reply:
AUTO is actually bad - I think we should change the default to WebAnno TSV 3. AUTO exports in the same format that was used for the import - that means, documents imported as plain text are exported as plain text - without any documentations. That is confusing for most users.

But anyway - for the purpose of just making a backup, the format does not matter at all because the archive always includes the SER files. So it would even make sense to make the export page clearer on that aspect, e.g. by having two boxes - one which is hard-coded to UIMA Serialized and then a "secondary export format" which the user can choose and which might even default to "none".

Reply:
So, I'll leave it for TSV since this is the format we genereally recommend. Users will be trained to prefer it.
ok?